Tests/BasicTestApp.Client/AutocompleteMultipleReadDataComponent.razor

```diff
@@ -32,15 +32,15 @@
     [Parameter]
     public List<string> SelectedTexts { get; set; }
 
-    private async Task OnHandleReadData( AutocompleteReadDataEventArgs autocompleteReadDataEventArgs )
+    private Task OnHandleReadData( AutocompleteReadDataEventArgs autocompleteReadDataEventArgs )
     {
         if ( !autocompleteReadDataEventArgs.CancellationToken.IsCancellationRequested )
         {
-            await Task.Delay( random.Next( 100 ) );
```

Comment:
How did we miss this??

Reply:
What do you mean?

Reply:
That's on a test component not the autocomplete itself.

Reply:
Ahhh. It was too early to look at the code. Ignore me :)